operator/config/scorecard/kustomization.yaml

```diff
@@ -1,16 +1,17 @@
 resources:
 - bases/config.yaml
-patchesJson6902:
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
```

Comment:
Move to the top?

Reply:
This is a result of `kustomize edit --fix`. Fixing this for better readability as suggested.